lmeval-scenarios.adoc

```diff
@@ -251,14 +253,19 @@ spec:
 
 == Using an InferenceService
 
-This example assumes that the vLLM model is already deployed in your cluster.
+In order to run an evaluation job on an InferenceService which is already deployed and running in your namespace, define your LMEvalJob CR, then apply this CR into the same namespace as your model.
```

Comment:
Similar comment as before, "In order" can be removed:

To run an evaluation job on an InferenceService that is already deployed and running in your namespace, define your LMEvalJob CR and then apply it to the same namespace as the model.